pkg/sentry/server/validator/kubernetes/kubernetes_test.go

```diff
@@ -795,3 +945,37 @@ func TestValidate(t *testing.T) {
 		})
 	}
 }
+
+func Test_isControlPlaneService(t *testing.T) {
```

Comment:
```suggestion
func isControlPlaneService(t *testing.T) {
```

IMO, a `test` prefix is duplicate in a test file.

Reply:
We need the `Test` prefix, otherwise the test won’t execute on `go test`